src/SFA.DAS.AANHub.Api/Controllers/EmployersController.cs

```diff
@@ -41,5 +42,30 @@ public async Task<IActionResult> CreateEmployer([FromHeader(Name = Constants.Pos
             : new BadRequestObjectResult(response.Errors);
 
         }
+
+        /// <summary>
+        /// Gets an employer member
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("{userId}")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(GetEmployerMemberResult), StatusCodes.Status200OK)]
+        public async Task<ActionResult> GetEmployer(long userId)
+        {
+            _logger.LogInformation("AAN Hub API: Received command to get employer by UserId: {userId}", userId);
+
+            var response = await _mediator.Send(new GetEmployerMemberQuery(userId));
+            if (response == null)
```

Comment:
I've just been doing similar in my local branch, and you might need to revisit the logic of this it won't fail, as the response will never be null, due to ValidatableResponse.

I've done something like this in my local branch, but it's a bit wordy, so I may need to address this in the middleware
```
if (response.Result == null && (response.Errors.Count == 0 || response.Errors == null)) return NotFound();
```
Mainly because that if it's a 404, then response.Errors will be of count 0. If it's a 400 request, response.Errors will be null.

Reply:
The response errors will be 0 when it is a success so how would you tell the difference between not-found and success?